### ISSUE TITLE
Version 2.4.0 does not honor setting "asciidoc.preview.useEditorStyle": false

### ISSUE THREAD

**Issue by gnschenker** [asciidoctor/asciidoctor-vscode#181]
After installing version `2.4.0` the setting `"asciidoc.preview.useEditorStyle": false` is ignored. When reinstalling version `2.3.3` it works again correctly.

**Comment by MatteoCampinoti94** [comment]
You are right, it's our fault.

When VSCode changed its API to handle extension reloading we solved the issue by removing contribution points, which contained the URIs of the stylesheets used by the preview.

Both asciidoctor.js and the Ruby version have their own stylesheet (the default one we used) so at least the preview is not completely broken.

It'll require some work to restore the editor style, but it should be doable without reworking the whole extension, we'll release a fix as soon as we can.

Sorry for the inconvenience :(

**Comment by MatteoCampinoti94** [comment]
The editor should actually use the default value (i.e. not the editor one), are you stuck with the editor style @gnschenker?

**Comment by eric-schneider** [comment]
@MatteoCampinoti94 I'm encountering the same issue. I have a dark theme for my editor, but I like the preview to have a light theme, so I just use the AsciiDoctor default theme (asciidoctor.css) since it acts as a good enough light theme. Here is what my setting looks like:

![image](https://user-images.githubusercontent.com/37347760/55364505-c1cfeb80-5495-11e9-9684-f234502c7f77.png)

However, the preview looks just like my editor now, which is not ideal on some of my monitors.

**Comment by MatteoCampinoti94** [comment]
@eric-schneider @gnschenker

Tey uninstalling the extension, restart VSCode, then reinstall it. That should hopefully fix the problem.

**Comment by MatteoCampinoti94** [comment]
I've looked deeper into the issues, and it might bot be easy to solve. We'll need to rewrite a portion of the extension and VSCode API for extensions changed a bit with update 1.31 to allow installing exrensions with no reload.
Id est there is no ETA on the fix, sorry.

**Comment by eric-schneider** [comment]
Turning off auto-update for extensions and downgrading to v2.3.3 fixed the issue. 

![image](https://user-images.githubusercontent.com/37347760/55368796-9dc9d580-54a8-11e9-9d0b-660d0158f5da.png)

![image](https://user-images.githubusercontent.com/37347760/55368800-a28e8980-54a8-11e9-9cdc-1979b9601d1c.png)

**Comment by eric-schneider** [comment]
I'm wondering if there's something to configure in the editor theme or settings.json that could be used as a workaround?

**Comment by MatteoCampinoti94** [comment]
@eric-schneider Thanks for that fix, downgrading is the best way at the moment.

Unfortunately there is no setting that will help. VSCode changed completely how extensions use "contribution points" (i.e. files that are used by the extension but are not strictly part of it, like our stylesheets). Now you need to declare them inside the extension and also advise the API that the extension requires a reload, or at least that's what I understood, alas the documentation to migrate to this new system is very lacking. On top of that I am no expert of TypeScript (Never liked the *Script languages to be honest, except maybe CoffeeScript).

So for now the fix seems to be:
1. Downgrade to 2.3.3
2. Set `asciidoc.preview.useEditorStyle` to `false`
3. Upgrade again to 2.4.0

Again, sorry for the problem, I'm studying it but I really do not know how long it will take to solve.

**Comment by MatteoCampinoti94** [comment]
@gnschenker @eric-schneider it's all fixed :3

The problem with the style being stuck was actually really easy to fix: the variable that pointed to the stylesheet directory was set to the wrong value, "assets" instead of "media". That and a weird if that was forgotten from an old version AND the change we made to abide with the API changes caused all the problems. Using the solution to that problem I also had the idea of passing the stylesheet as option directly to the processors together with the installed location of the extension, this way I got around the contribution points ^.^

So the `useEditorStyle` option now works as intended, even better than before actually!

I'm currently working on the editor-style css to match it with the default one, once that is done we'll push the new version out :)

Thank you for your patience!
